web/validation.go

```diff
@@ -219,12 +219,15 @@ func ValidateForm(guild *dstate.GuildSet, tmpl TemplateData, form interface{}) b
 	return ok
 }
 
-func readMinMax(valid *ValidationTag) (float64, float64) {
-
+func readMinMax(valid *ValidationTag) (float64, float64, bool) {
+	onlyMin := false
 	min, _ := valid.Float(0)
 	max, _ := valid.Float(1)
+	if valid.values[1] == "" {
+		onlyMin = true
+	}
 
-	return min, max
+	return min, max, onlyMin
```

Comment:
It's possible to simplify this thusly: `return min, max, valid.values[1] == ""`. Whether this is a good idea is debatable, but figured I'd throw it out there.